### DISCUSSION TITLE
mountain duck slow download compared to cyberduck

### DISCUSSION THREAD

**Discussion by openaspace** [iterate-ch/cyberduck#15372] [Q&A]
Hello I see that mountainduck download slowly compared to cyberduck .. for the same server and same file download.. it's multipart download upload enabled in mountainduck?

**Comment by openaspace** [comment]
it download at 18MB/s instead of cyberduck at 78 MB/s

**Comment by dkocher** [comment]
You should see comparable download rates if you choose _Keep Offline_ for a file [^1]. When copying or opening files in Finder.app or Windows Explorer the download is limited to a single connection.

[^1]: https://docs.mountainduck.io/mountainduck/connect/sync/#keep-offline

**Comment by openaspace** [comment]
thank you, but i can't do this, in the sense.. i need the download with multi part , are folder of 250 gb or more, i can't use the sync.
Why don't enable multipart download?
